### DISCUSSION TITLE
Custom Title Bar

### DISCUSSION THREAD

**Discussion by jimgilliam** [microsoft/win-student-devs#99] [Q&A]
When I put the display_override in my manifest file and run my PWA, I do not see an empty space.... I see the default title bar. I've followed the instructions, but I must still be doing something wrong. Advice?

**Comment by captainbrosset** [comment]
The only thing that comes to mind is that, as a user, you have to click the **Hide title bar** icon (`^`) in the title bar for the feature to be enabled. The app can signal that it supports it, but ultimately, it's a user choice if they want to use it.

If you did click the icon already and it doesn't work, or if the icon doesn't appear, then it's a different issue than I thought. In which case, it would be great if you could provide the source code to your app, or a link to it, so we can try and investigate the problem.

**Comment by jimgilliam** [comment]
@captainbrosset Captain, it's working now after adjustments to my manifest. Thank you.
